app/src/main/java/com/example/peterchu/watplanner/Models/Schedule/CourseComponent.java

```diff
@@ -281,15 +281,15 @@ public Calendar getCalendarEndTime() {
 
                     event.setStartTime(eventStartDate);
                     event.setEndTime(eventEndDate);
-                    event.setLocation(location.toString());
+                    event.setLocation(type);
```

Comment:
oh nvm, ignore this. It looks better actually to not show location info in the name of the event